translations/ja.json

```
@@ -298,58 +298,58 @@
 	"Generating certificates and keys ...": "証明書と鍵を作成しています...",
-	"Get or list the current profiles (clusters)": "現在指定しているクラスタプロファイルを取得、またはリストアップします",
-	"Gets the logs of the running instance, used for debugging minikube, not user code.": "",
-	"Gets the status of a local Kubernetes cluster": "ローカル Kubernetes クラスタの状態を取得します",
-	"Gets the status of a local Kubernetes cluster.\n\tExit status contains the status of minikube's VM, cluster and Kubernetes encoded on it's bits in this order from right to left.\n\tEg: 7 meaning: 1 (for minikube NOK) + 2 (for cluster NOK) + 4 (for Kubernetes NOK)": "",
-	"Gets the value of PROPERTY_NAME from the minikube config file": "",
+	"Get or list the current profiles (clusters)": "現在のプロファイル (クラスター) を取得または一覧表示します",
+	"Gets the logs of the running instance, used for debugging minikube, not user code.": "実行中のインスタンスのログを取得します (ユーザーコードではなく minikube デバッグに使用)",
+	"Gets the status of a local Kubernetes cluster": "ローカル Kubernetes クラスターの状態を取得します",
+	"Gets the status of a local Kubernetes cluster.\n\tExit status contains the status of minikube's VM, cluster and Kubernetes encoded on it's bits in this order from right to left.\n\tEg: 7 meaning: 1 (for minikube NOK) + 2 (for cluster NOK) + 4 (for Kubernetes NOK)": "ローカル Kubernetes クラスターの状態を取得します。\n\t終了ステータスは minikube の VM、クラスター、Kubernetes の状態を順に右→左のビット列でエンコードしたものを含みます。\n\t例: 7 = 1 (mknikube 異常) + 2 (クラスター異常) + 4 (Kubernetes 異常)",
+	"Gets the value of PROPERTY_NAME from the minikube config file": "minikube 設定ファイル中の PROPERTY_NAME の値を取得します",
 	"Global Flags": "グローバルなフラグ",
-	"Go template format string for the cache list output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#CacheListTemplate": "",
-	"Go template format string for the config view output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd/config#ConfigViewTemplate": "",
-	"Go template format string for the status output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status": "",
+	"Go template format string for the cache list output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#CacheListTemplate": "キャッシュ一覧出力用の Go テンプレートフォーマット文字列。Go テンプレートのフォーマットはこちら: https://golang.org/pkg/text/template/\nテンプレートでアクセス可能な変数の一覧は、こちらの構造化変数を参照してください: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status",
```

```suggestion
	"Go template format string for the cache list output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#CacheListTemplate": "キャッシュ一覧出力用の Go テンプレートフォーマット文字列。Go テンプレートのフォーマットはこちら: https://golang.org/pkg/text/template/\nテンプレートでアクセス可能な変数の一覧は、こちらの構造化変数を参照してください: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#CacheListTemplate",
```
It's different from original URL for variables we can use.